addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java

```diff
@@ -108,6 +108,7 @@ public AddAscanRuleDialog(
         this.addComboField(STRENGTH_PARAM, allstrengths, strengthName);
 
         this.addPadding();
+        pack();
```

Comment:
Might worth leaving a comment in the constructor saying that the dimension being passed will not make any difference.

Reply:
Should I just remove the dimension?

Reply:
Disregard, all of the constructors expect one.

Reply:
That would lead to NPE.